exo-run/bin/features

```diff
@@ -2,6 +2,8 @@
 set -e
 
 node_modules/o-tools-livescript/bin/build
+cd ../exo-setup && bin/setup && node_modules/.bin/build
```

Comment:
Run this line inside a subshell, to avoid having to go back. Changing directory in the subshell doesn't affect the current directory in the main shell.

```bash
(cd ../exo-setup && bin/setup && node_modules/.bin/build)
if [ "$#" == "0" ]; then